### DISCUSSION TITLE
Messages in multiplayer are only polled once per minute

### DISCUSSION THREAD

**Discussion by PandaDriver156** [ppy/osu#16437] [Q&A] [answered]
Messages sent in a lobby chat can take up to a minute to show up to other players.
Happens since the latest 2022.108.0 release.

Steps to reproduce:
  1. Enter a lobby with 2 clients
  2. Send a message on a client
  3. Message will not show up on the other client for around a minute, except if chat overlay is opened.

This very much seems to be caused by #16253, as opening the chat overlay does make messages sent in the lobby appear immediately.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/16438
